addons/base_iban/static/src/tests/helpers/model_definitions_setup.js

```diff
@@ -0,0 +1,5 @@
+/** @odoo-module **/
+
+import { addModelNamesToFetch } from '@bus/../tests/helpers/model_definitions_helpers';
+
+addModelNamesToFetch(['res.partner.bank']);
```

Comment:
@h4818  I found out that this is why you were having issues when trying to create the test with pyenv.
Those lines are responsible for making JS fetch the models informations